lib/Drawing/Color.vala

```diff
@@ -103,6 +143,27 @@ namespace Granite.Drawing {
             set_from_rgba (rgba);
         }
 
+        /**
+         * Conststructs a new {@link Granite.Drawing.Color} from an integer.
+         * 
+         * This constructor should be used when deserializing the previously serialized
+         * color by {@link Granite.Drawing.Color.to_int}.
+         * 
+         * For more details on what format the color integer representation has, see {@link Granite.Drawing.Color.to_int}.
+         * 
+         * If you would like to deserialize the A, R, G and B values from the integer without
+         * creating a new instance of {@link Granite.Drawing.Color}, you can use the available
+         * //*_from_int// static method collection such as {@link Granite.Drawing.Color.alpha_from_int}.
+         * 
+         * @param color the integer specyfying the color
+         */
+        public Color.from_int (int color) {
+            R = (double)red_from_int (color) / uint8.MAX;
+            G = (double)green_from_int (color) / uint8.MAX;
+            B = (double)blue_from_int (color) / uint8.MAX;
+            A = (double)alpha_from_int (color) / uint8.MAX;
+        }
```

Comment:
Will Vala always automatically do the correct conversions? Else should be:
 `(double)red)_from_int (color) / (double)uint8.MAX`
 etc.